README.md

```diff
@@ -37,29 +38,96 @@ Decorators and some other features for sequelize (v5).
  - [Recommendations and limitations](#recommendations-and-limitations)
 
 ## Installation
-*sequelize-typescript* requires [sequelize](https://github.com/sequelize/sequelize), additional typings as documented [here](https://docs.sequelizejs.com/manual/typescript.html) and [reflect-metadata](https://www.npmjs.com/package/reflect-metadata)
+
+- this assumes usage of `sequelize@6`
+- *sequelize-typescript* requires [sequelize](https://github.com/sequelize/sequelize)
+- additional typings as documented [here](https://sequelize.org/master/manual/typescript.html) and [reflect-metadata](https://www.npmjs.com/package/reflect-metadata)
+
 ```
 npm install sequelize
-npm install @types/bluebird @types/node @types/validator
+npm install @types/node @types/validator
 npm install reflect-metadata
 ```
+
 ```
 npm install sequelize-typescript
 ```
+
 Your `tsconfig.json` needs the following flags:
+
 ```json
 "target": "es6", // or a more recent ecmascript version
 "experimentalDecorators": true,
 "emitDecoratorMetadata": true
 ```
 
+### ⚠️ sequelize@5
+`sequelize@5` requires `sequelize-typescript@1`. See
+[documentation](https://github.com/RobinBuschmann/sequelize-typescript/tree/1.0.0) for version `1.0`.
+```
+npm install sequelize-typescript@1.0
+```
+
 ### ⚠️ sequelize@4
 `sequelize@4` requires `sequelize-typescript@0.6`. See
 [documentation](https://github.com/RobinBuschmann/sequelize-typescript/tree/0.6.X) for version `0.6`.
 ```
 npm install sequelize-typescript@0.6
 ```
 
+## Upgrade to `sequelize-typescript@2`
+
+- `sequelize-typescript@2` only works with `sequelize@6.2>=`.
+For `sequelize@5` use `sequelize-typescript@1.0`.
+
+### Breaking Changes
+
+- All breaking changes of `sequelize@6` are also valid for `sequelize-typescript@2`.
+See [Upgrade to v6](https://sequelize.org/master/manual/upgrade-to-v6.html) for details.
+- `@types/bluebird` is no longer needed, `sequelize@6` removed usage of `bluebird`
+- Sequelize v6.2 introduced additional model attributes typings, which affects how the model is defined.
+- See below comparison between V5 and V6 model definition to show how to upgrade models.
+- For more details, see [sequelize typescript docs](https://sequelize.org/master/manual/typescript.html).
+
+#### V5 Model definition
+
+```typescript
+import { Table, Model } from 'sequelize-typescript';
+
+@Table
+class Person extends Model<Person> {}
+```
+
+#### V6 Model definition (less strict)
+
+```typescript
+import { Table, Model } from 'sequelize-typescript';
+
+@Table
+class Person extends Model {}
+```
+
+#### V6 Model definition (more strict)
+
+- to allow more strict model attributes type-checks, you can define `ModelAttributes` and `ModelCreationAttributes` interfaces
+
+
+```typescript
+import { Table, Model } from 'sequelize-typescript';
+
+interface PersonAttributes {
+  id: number;
+  name: string;
+}
+
+interface PersonCreationAttributes {
+  name: string;
+}
```

Comment:
```suggestion
interface PersonCreationAttributes extends Omit<PersonAttributes, 'id'> {
}
```

Reply:
Thanks @intellix, added with corresponding import statement. Good idea to keep it consistent with sequelize official TS guide.
...though anyone can freely to construct the interfaces however feels appropriate.